### PR TITLE
README: re-organize the sections.

### PR DESCRIPTION
**Pull request by songy23**
Fixes https://github.com/census-instrumentation/opencensus-service/issues/54.

As I read through README, I found most of the old contents (especially the initiatives) are still relevant, so I just slightly modified the order of sections.

/cc @rakyll

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# OpenCensus service
+# OpenCensus Service
 
-OpenCensus service is an experimental component that collects and
+OpenCensus Service is an experimental component that collects and
 exports from the OpenCensus instrumented processes available from the
```
Comment (bogdandrutu) [inline README.md]: Mention the support for other formats, as well as the fact that we support multiple signals (metrics).

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 same host machine.
```
Comment (bogdandrutu) [inline README.md]: This is not true anymore.

File: README.md [hunk 3]
```diff
@@ -13,7 +13,9 @@ with OpenCensus, they have to modify their code to register an
 exporter and redeploy. Asking our users recompile and redeploy is
```
Comment (bogdandrutu) [inline README.md]: Also another case is if somebody wants to distribute a binary instrumented with OC which export should they use, with the agent they can distribute the binary with the agent exporter and the user will run the agent with the exporter of choice.

File: README.md [hunk 4]
```diff
@@ -11,9 +13,18 @@ during an incident. Even though our users can benefit from having
...
 with OpenCensus, they have to modify their code to register an
 exporter and redeploy. Asking our users recompile and redeploy is
-not an ideal at an incident time.
+not an ideal at an incident time. In addition, currently users need
+to decide which service backend they want to export to, before they
+distribute their binary instrumented by OpenCensus.
 
-OpenCensus service is trying to eliminate this requirement.
+OpenCensus Service is trying to eliminate these requirements. With
+OpenCensus Service, users do not need to redeploy or restart their binary
```
Comment (odeke-em) [inline README.md]: restart their applications

File: README.md [hunk 5]
```diff
@@ -11,9 +13,18 @@ during an incident. Even though our users can benefit from having
...
+as long as it has the OpenCensus Agent exporter. All they need to do is
+just configuring and deploying OpenCensus Service, then it will automatically
```
Comment (odeke-em) [inline README.md]: just configure and deploy the OpenCensus service separately. OpenCensus service will then
automatically collect traces...

**Pull request merged**